src/server/journal/tx_executor.cc

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024, DragonflyDB authors.  All rights reserved.
```

Comment:
Are there changes in the file which are not simple cut-paste?

Reply:
a couple of lines I've added one parameter to  a InsertTxToSharedMap method